packages/frontend/pages/index.tsx

```diff
@@ -13,9 +16,23 @@ const tempAutoConnectConfig: TAutoConnect = {
 };
 
 const Home: NextPage = () => {
+  const [price, setPrice] = useState<number>(0);
+  const mainnetProvider = useProvider({ chainId: 1 });
```

Comment:
This should be inside the hook. We always want to get the ETH price from mainnet.